pages/account/manage/testimonials.js

```diff
@@ -53,7 +53,7 @@ export default function ManageTestimonials({ BASE_URL, testimonials }) {
   const [showNotification, setShowNotification] = useState(false);
   const [testimonialList, setTestimonialList] = useState(testimonials || []);
   const [testimonialListPrevious, setTestimonialListPrevious] = useState(
-    testimonials || []
+    testimonials || [],
```

Comment:
Hi Dan

My guess is prettier because that is the only formatting extension I have. 

What do you advise?

Reply:
Disable your prettier extension.
VSCode lets you do this for only the current workspace, so it will only effect this project.
You will need to go in & manually revert the changes it made.

Reply:
Thank you.

I created another branch demo-2 and push from there.

Reply:
You are better off making the changes in your current branch.

There is no way to change the source branch for this PR. You would have to close this PR & open a new one.

Reply:
Ok

Thanks